src/utils/utils.tsx

```diff
@@ -79,3 +79,28 @@ export const NumberDateToWordDate = (date_str: string) => {
   const monthStr = months[Number(month) - 1];
   return `${Translate("Months", [monthStr])} ${day}, ${year}`
 }
+
+export const getformattedDate = (day: string | undefined, month: string | undefined, year: string | undefined, language: LanguageType) => {

```

Comment:
might be simpler to use a switch statement here, something like:

`
switch(language){   
        case 'en': return `${month } ${day ? day + "," : ""} ${year}`;
        case 'fr': return `${day} ${Translate("Months", [month])} ${year}`;
        case 'de': return `${day ? day + "." : ""} ${Translate("Months", [month])} ${year}`;
        default: return `${month } ${day ? day + "," : ""} ${year}`;      
    }
`